facia-purger/src/main/scala/com/gu/purge/facia/ParseS3Path.scala

```diff
@@ -0,0 +1,21 @@
+package com.gu.purge.facia
+
+import org.parboiled2.{Parser, ParserInput}
+
+import scala.util.Success
+
+// /aws-frontend-store/CODE/frontsapi/pressed/live/au/fapi/pressed.json
+class ParseS3Path(stage: String, val input: ParserInput) extends Parser {
```

Comment:
you can also make this a `case class` so you get the apply constructor for free among other things and you won't have to use `new ParseS3Path(...)` but only `ParseS3Path(...)`